changes for building with gnome-desktop 44.0

drop when updating eog to a newer version

--- a/src/eog-thumbnail.c	2024-01-20 21:25:56.395811874 -0800
+++ b/src/eog-thumbnail.c	2024-01-20 21:29:36.144323091 -0800
@@ -505,16 +505,16 @@
 		} else {
 			/* generate a thumbnail from the file */
 			eog_debug_message (DEBUG_THUMBNAIL, "%s: creating from file",data->uri_str);
-			thumb = gnome_desktop_thumbnail_factory_generate_thumbnail (factory, data->uri_str, data->mime_type);
+			thumb = gnome_desktop_thumbnail_factory_generate_thumbnail (factory, data->uri_str, data->mime_type, NULL, NULL);
 		}
 
 		if (thumb != NULL) {
 			/* Save the new thumbnail */
-			gnome_desktop_thumbnail_factory_save_thumbnail (factory, thumb, data->uri_str, data->mtime);
+			gnome_desktop_thumbnail_factory_save_thumbnail (factory, thumb, data->uri_str, data->mtime, NULL, NULL);
 			eog_debug_message (DEBUG_THUMBNAIL, "%s: normal thumbnail saved",data->uri_str);
 		} else {
 			/* Save a failed thumbnail, to stop further thumbnail attempts */
-			gnome_desktop_thumbnail_factory_create_failed_thumbnail (factory, data->uri_str, data->mtime);
+			gnome_desktop_thumbnail_factory_create_failed_thumbnail (factory, data->uri_str, data->mtime, NULL, NULL);
 			eog_debug_message (DEBUG_THUMBNAIL, "%s: failed thumbnail saved",data->uri_str);
 			set_thumb_error (error, EOG_THUMB_ERROR_GENERIC, "Thumbnail creation failed");
 		}
--- a/src/eog-file-chooser.c	2024-01-20 21:25:09.664865987 -0800
+++ b/src/eog-file-chooser.c	2024-01-20 21:25:32.454085108 -0800
@@ -375,7 +375,7 @@
 
 				if (G_LIKELY (can_thumbnail && !has_failed))
 					pixbuf = gnome_desktop_thumbnail_factory_generate_thumbnail (
-							priv->thumb_factory, uri, mime_type);
+							priv->thumb_factory, uri, mime_type, NULL, NULL);
 
 				g_free (mime_type);
 			}
